components/waitwhile/actions/retrieve-customer/retrieve-customer.ts

```diff
@@ -0,0 +1,25 @@
+import waitwhile from "../../app/waitwhile.app";
+import { defineAction } from "@pipedream/types";
+
+
+export default defineAction({
+  name: "Retrieve Customer",
+  version: "0.0.2",
```

Comment:
Should be version 0.0.1
```suggestion
  version: "0.0.1",
```